.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+##### What version (commit) do you encounter this issue on?
+
+##### What type of issue is this (bug, feature, documentation, regression)?
+
+##### What is the expected behavior?
+
+##### What is actually happening?
+
+##### How can we reproduce this issue?
+
+##### How can we verify your issue is addressed?
+
+##### If this is a large issue, what can be done immediately to alleviate the issue while working on a longer-term solution?
+
+##### What is your evaluation on the importance of this issue, and why?
+
+##### What branches / milestones should this issue target?
+The list below is what is currently supported for backports, remove branches you do not wish to target.
+- [ ] Master
+- [ ] liberty-12.0
+- [ ] kilo
```

Comment:
I guess it's just like someone in launchpad adding *-backport-potential tags - they can always be removed later by the triage/reviewer/maintainer team. So yeah, maybe a tweak to the wording but other than that looks good.